packages/graphics/src/GraphicsGeometry.js

```diff
@@ -215,9 +215,11 @@ export default class GraphicsGeometry extends BatchGeometry
             this.colors.length = 0;
             this.uvs.length = 0;
             this.indices.length = 0;
+            this.textureIds.length = 0;
 
-            for (let i = 0; i < DRAW_CALL_POOL.length; i++)
+            for (let i = 0; i < this.drawCalls.length; i++)
```

Comment:
so we have global `DRAW_CALL_POOL` and local `this.drawCalls`, would be nice to add some doc what is the purpose for duplicated `DRAW_CALL_POOL`